tools/test-backend

```diff
@@ -22,123 +22,132 @@ import ujson
-    'zerver/lib/import_realm.py',
-    'zerver/lib/logging_util.py',
-    'zerver/lib/migrate.py',
-    'zerver/lib/outgoing_webhook.py',
-    'zerver/lib/parallel.py',
-    'zerver/lib/profile.py',
-    'zerver/lib/queue.py',
-    'zerver/lib/sqlalchemy_utils.py',
-    'zerver/lib/storage.py',
-    'zerver/lib/timeout.py',
-    'zerver/lib/unminify.py',
-    'zerver/lib/utils.py',
-    'zerver/lib/zephyr.py',
-    'zerver/templatetags/app_filters.py',
-    'zerver/templatetags/minified_js.py',
-    # Low priority for coverage
-    'zerver/lib/ccache.py',
-    'zerver/lib/generate_test_data.py',
-    'zerver/lib/server_initialization.py',
-    'zerver/lib/test_fixtures.py',
-    'zerver/lib/test_runner.py',
-    'zerver/openapi/python_examples.py',
-    # Tornado should ideally have full coverage, but we're not there.
-    'zerver/tornado/autoreload.py',
-    'zerver/tornado/descriptors.py',
-    'zerver/tornado/event_queue.py',
-    'zerver/tornado/exceptions.py',
-    'zerver/tornado/handlers.py',
-    'zerver/tornado/ioloop_logging.py',
-    'zerver/tornado/sharding.py',
-    'zerver/tornado/views.py',
-    # Data import files; relatively low priority
-    'zerver/data_import/hipchat*.py',
-    'zerver/data_import/sequencer.py',
-    'zerver/data_import/slack.py',
-    'zerver/data_import/gitter.py',
-    'zerver/data_import/import_util.py',
-    # Webhook integrations with incomplete coverage
-    'zerver/webhooks/greenhouse/view.py',
-    'zerver/webhooks/jira/view.py',
-    'zerver/webhooks/librato/view.py',
-    'zerver/webhooks/pivotal/view.py',
-    'zerver/webhooks/semaphore/view.py',
-    'zerver/webhooks/solano/view.py',
-    'zerver/webhooks/teamcity/view.py',
-    'zerver/webhooks/travis/view.py',
-    'zerver/webhooks/zapier/view.py',
-] for path in glob.glob(target)}
+target_fully_covered = {
+    path
+    for target in [
+        "analytics/lib/*.py",
+        "analytics/models.py",
+        "analytics/tests/*.py",
+        "analytics/views.py",
+        # zerver/ and zerver/lib/ are important core files
+        "zerver/*.py",
+        "zerver/lib/*.py",
+        "zerver/lib/*/*.py",
+        "zerver/lib/*/*/*.py",
+        "zerver/data_import/*.py",
+        "zerver/templatetags/*.py",
+        "zerver/tornado/*.py",
+        # Billing files require 100% test coverage
+        "corporate/lib/stripe.py",
+        "corporate/views.py",
+        # Test files should have 100% coverage; test code that isn't run
+        # is likely a bug in the test.
+        "zerver/tests/*.py",
+        "corporate/tests/*.py",
+        # As a project, we require 100% test coverage in the views files.
+        "zerver/views/*.py",
+        "zproject/backends.py",
+        "confirmation/*.py",
+        "zerver/webhooks/*/*.py",
+        # Once we have a nice negative tests system, we can add these:
+        # 'zerver/webhooks/*/*.py',
+        # 'zerver/webhooks/*/*/*.py',
+        "zerver/worker/*.py",
+    ]
+    for path in glob.glob(target)
+}
```

Comment:
These might benefit from extracting another variable to avoid Black adding an extra layer of indentionation around the globs?